.github/workflows/ci-nym-vpn-cli.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-22.04-arc, custom-runner-mac-m1, windows-2022 ]
+        os: [ ubuntu-22.04, custom-runner-mac-m1, windows-2022 ]
```

Comment:
Doesn't it work with the arc runner?

Reply:
Oh, I'm not sure when or how that got removed 😄

Reply:
I guess something is wrong with arc so I am switching it back.

Reply:
It failed because of the change further down, the arc runner should be fine